### PR TITLE
Polish student apps

### PR DESCRIPTION
**Pull request by wonjun**

### CODE REVIEW

**Comment by wonjun** [comment]
Hm, this isn't failing locally! Could you check to see what it says on codeship? @SamLau95

**Comment by SamLau95** [comment]
* student_application - missing required :bucket option (ArgumentError)

is the error!

**Comment by wonjun** [comment]
@SamLau95 do you think you could double check the aws keys and the bucket are set on the test environment too?

Also, could you add me to codeship?

File: spec/features/student_application_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "Student Application Form" do
+  before do
+    visit new_student_application_path
+    expect(page).to have_content "You need to sign in"
+    create_omniauth
+    click_link 'Sign in with Facebook'
+    visit new_student_application_path
+  end
```
Comment (SamLau95) [inline spec/features/student_application_spec.rb]: Nice! You got everything to work :)

**Comment by SamLau95** [comment]
Oh! I think the reason why the env variables aren't getting set on codeship is because you didn't put in dummy values in the application.yml.sample . Try doing that and seeing if it fixes things.

**Comment by wonjun** [comment]
Ok, so in my rspec tests I actually upload a file to S3. Maybe that's not the best way to go about it? 

To do that, I had to add the environment variable keys to codeship

**Comment by SamLau95** [comment]
Ohh you definitely should not do that hahaha since we might run the tests
many times (esp with guard). We should instead mock the uploading so we
don't hit amazon.

**Comment by wonjun** [comment]
Ok! I kept the uploading in the feature integration test, is that fine? I did mock it for factories

**Comment by wonjun** [comment]
Ok so I changed it so paperclip uses s3 only on production.

**Comment by SamLau95** [comment]
Looks good to me! But merge conflicts @wonjun :(

**Comment by SamLau95** [comment]
Yay?

**Pull request merged**